src/core/Tools.cpp

```diff
@@ -240,4 +246,110 @@ void setupSearchPaths()
 #endif
 }
 
+//
+// Prevent memory dumps without admin privileges.
+// MiniDumpWriteDump function requires PROCESS_QUERY_INFORMATION and PROCESS_VM_READ
+// see: https://msdn.microsoft.com/en-us/library/windows/desktop/ms680360%28v=vs.85%29.aspx
+//
```

Comment:
I will add a more detailed comment tomorrow.
Please verify that it's working with windows 10 too.